modules/proj4/docs/api-reference/proj4-projection.md

```diff
@@ -1,13 +1,15 @@
 # Ellipsoid
 
-
 ## Usage
 
-Reproject WGS84 coordinates in another CRS
+Reproject WGS84 coordinates to another CRS
```

Comment:
From CRS to CRS not just WGS84

Reply:
Here the example is from WGS84, I thought it would be clear enough to a user

Reply:
Ah yes, makes sense, please disregard my comment.